server/neptune/workflows/activities/github/markdown/renderer.go

```diff
@@ -4,17 +4,33 @@ import (
 	"bytes"
 	_ "embed" //embedding files
 	"fmt"
-	"github.com/runatlantis/atlantis/server/neptune/workflows/activities/terraform"
 	"html/template"
 
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities/github"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities/terraform"
+
 	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/terraform/state"
 )
 
+//go:embed templates/planconfirm.tmpl
+var planConfirmStr string
+
 //go:embed templates/checkrun.tmpl
 var checkrunTemplateStr string
 
 // panics if we can't read the template
 var checkrunTemplate = template.Must(template.New("").Parse(checkrunTemplateStr))
+var planConfirmTemplate = template.Must(template.New("").Parse(planConfirmStr))
+
+type planconfirmTemplateData struct {
+	Revision              string
+	RevisionURL           string
+	Pull                  int
+	PullURL               string
```

Comment:
I don't think these are populated, will they be later on?

Reply:
read my PR description

Reply:
ahh missed that, approved!